uint8 MsgType
uint8 Enable
uint16 Interval
---